src/lib/utils/FailoverProvider.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why? i don't see anywhere you couldn't just use the safer `unknown` instead of `any` in this file